.github/renovate.json

```diff
@@ -22,6 +22,10 @@
     {
       "matchManagers": ["github-actions"],
       "groupName": "GitHub Actions"
+    },
+    {
+      "matchPackageNames": ["node-fetch"],
+      "allowedVersions": "<3.0.0"
```

Comment:
We might want to include `yn < 5` as well: https://github.com/backstage/community-plugins/pull/468

Reply:
good catch, added `yn < 5`!